yaramo/topology.py

```diff
@@ -31,3 +32,21 @@ def get_edge_by_nodes(self, node_a: Node, node_b: Node):
                 return edge
         return None
 
+    def to_serializable(self):
```

Comment:
If Topology inherits `BaseElement`, it is a bit confusing that its `to_serializable` method has a different signature.